lint.swift

```diff
@@ -0,0 +1,547 @@
+        regex: [
+            "openingBrace": #"\("#,
+            "callPart1": #"[^\s\?\.]+"#,
+            "separator1": #"\?\."#,
+            "callPart2": #"[^\s\?\.]+"#,
+            "closingBraceUnwrap": #"\)!"#
+        ],
+        matchingExamples: ["call(x: (viewModel?.username)!)"],
+        nonMatchingExamples: ["call(x: (viewModel?.profile?.username)!)", "call(x: viewModel!.username)"],
+        includeFilters: [swiftFiles],
+        excludeFilters: [testFiles],
+        autoCorrectReplacement: "$callPart1!.$callPart2",
+        autoCorrectExamples: [
+            ["before": "call(x: (viewModel?.username)!)", "after": "call(x: viewModel!.username)"]
+        ]
+    )
+
+    // MARK: MultilineIfStructure
+    try Lint.checkFileContents(
+        checkInfo: "MultilineIfStructure: Make sure multiline if conditions are all on their own lines, indented by four spaces.",
+        regex: #"^( *)if\s*(.*,) *\n((?:.+, *\n)*) *(.*\S)\s*\{\s*"#,
+        includeFilters: [swiftFiles],
+        excludeFilters: [testFiles],
+        autoCorrectReplacement: "$1if\n$1    $2\n$3$1    $4\n$1{\n$1    $5",
+        autoCorrectExamples: [
+            [
+                "before": """
+                    if let employee = UserDefaults.standard.currentEmployee(),
+                        let employeeName = employeeName,
+                        let employeeEmail = employeeEmail,
+                        let recipientEmail = recipientEmail {
+                            dataExporterHelper.exportData(
+                """,
+                "after": """
+                    if
+                        let employee = UserDefaults.standard.currentEmployee(),
+                        let employeeName = employeeName,
+                        let employeeEmail = employeeEmail,
+                        let recipientEmail = recipientEmail
+                    {
+                        dataExporterHelper.exportData(
+                """
+            ],
+            [
+                "before": """
+                    if let employee = UserDefaults.standard.currentEmployee(),
+                        let recipientEmail = recipientEmail
+                    {
+                        dataExporterHelper.exportData(
+                """,
+                "after": """
+                    if
+                        let employee = UserDefaults.standard.currentEmployee(),
+                        let recipientEmail = recipientEmail
+                    {
+                        dataExporterHelper.exportData(
+                """
+            ]
+        ]
+    )
+
+    // MARK: MultilineGuardEnd
+    try Lint.checkFileContents(
+        checkInfo: "MultilineGuardEnd: Always close a multiline guard via `else {` on a new line indented like the opening `guard`.",
+        regex: #"guard\s*([^\n]*,\n)+([^\n]*\S *)else\s*\{"#,
+        matchingExamples: [
+            """
+            guard
```

Comment:
Codacy found an issue: [Always close a multiline guard via `else {` on a new line indented like the opening `guard`.](https://app.codacy.com/gh/Flinesoft/HandySwift/pullRequest?prid=5730214)